spec/models/like_spec.rb

```diff
@@ -0,0 +1,45 @@
+require_relative '../rails_helper'
+
+RSpec.describe Like, type: :model do
+  let(:user) do
+    User.new(
+      name: 'Love',
+      photo: 'https://unsplash.com/photos/4aQMAif1ZKk',
+      bio: 'I am a model',
+      posts_counter: 4
+    )
+  end
+
+  let(:post) do
+    Post.new(
+      users: user,
+      title: 'My first photoshoot',
+      text: 'This is my first photoshoot',
+      comments_counter: 1,
+      likes_counter: 2
+    )
+  end
+
+  let(:comment) do
+    Comment.new(
+      users: user,
+      posts: post,
+      text: 'Hey!, it is my first photoshoot'
+    )
+  end
+
+  it 'only valid with a user' do
```

Comment:
- [ ] Some controller specs are failing. Kindly ensure that **all** test cases are passing.

**Failing RSpec tests:**
![image](https://user-images.githubusercontent.com/12745474/195997049-dcca7df1-cf75-4bcd-b319-65691d0cc0e9.png)